package.json

```diff
@@ -35,7 +37,9 @@
     "test": "jest --env=jsdom"
   },
   "dependencies": {
+    "dotenv-webpack": "^8.0.1",
```

Comment:
Kindly remove all packages you are not using from you project. This will enable faster download and installation in the future.
 It also reduces the chances of depending on packages that are poorly maintained and  which will thus introduce security holes.